torch/distributed/_composable/fsdp/_fsdp_common.py

```diff
@@ -49,3 +51,11 @@ class HSDPMeshInfo(FSDPMeshInfo, DDPMeshInfo):
     def __post_init__(self):
         super(FSDPMeshInfo, self).__post_init__()
         super(DDPMeshInfo, self).__post_init__()
+
+
+def _is_composable_with_fsdp(module: nn.Module) -> bool:
+    registry = _get_registry(module)
+    if registry is None:
+        return True
+    # TODO: Add the TorchRec composable API name.
+    return "replicate" not in registry
```

Comment:
To confirm, this means FSDP2 would not step into `replicate` submodule, but it would step into all other submodules?

Reply:
Yep!